client/src/components/NavBar.jsx

```diff
@@ -35,8 +35,10 @@ export default function (props) {
   return (
     <Box className={classes.header}>
       {auth.isAuthenticated() ? <Button
-        className={`${classes.invisible} ${classes.navSides}`}>
-        Sign out
+        variant="contained"
+        color="primary"
+        onClick={() => { history.push("/welcome") }}>
+        New Game
```

Comment:
Just a heads up, I'm also adding a new game button in navbar - but mine will be a bit below this.

Reply:
great, thank you. You can overwrite it when you merge yours then